--- ./a/install.sh	2014-12-21 16:54:41.000000000 +0900
+++ ./b/install.sh	2014-12-22 02:39:24.376676682 +0900
@@ -370,18 +370,6 @@
     fi
 fi

-# Sanity check: can we can write to the destination?
-msg "verifying destination is writable"
-umask 022 && mkdir -p "${CFG_LIBDIR}"
-need_ok "can't write to destination. consider \`sudo\`."
-touch "${CFG_LIBDIR}/rust-install-probe" > /dev/null
-if [ $? -ne 0 ]
-then
-    err "can't write to destination. consider \`sudo\`."
-fi
-rm -f "${CFG_LIBDIR}/rust-install-probe"
-need_ok "failed to remove install probe"
-
 # Sanity check: don't install to the directory containing the installer.
 # That would surely cause chaos.
 msg "verifying destination is not the same as source"